documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -105,3 +112,17 @@ kubectl create secret generic _CA-KEY-SECRET_ --from-file=ca.key=_CA-KEY-SECRET-
 kubectl label secret _CA-CERTIFICATE-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_KAFKA-CLUSTER-NAME_
 kubectl label secret _CA-KEY-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_KAFKA-CLUSTER-NAME_
 ----
+
+. Annotate the secrets with the generation annotations:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-CERTIFICATE-SECRET_ strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_
+kubectl annotate secret _CA-KEY-SECRET_ strimzi.io/ca-key-generation=_CA-KEY-GENERATION_
```

Comment:
What happens when reconciliation hits between updating the secret and updating the annotations? Shouldn't this instead of using `kubectl label` and `kubectl annotate` work with editing the original secret?